src/Idris/AbsSyntax.hs

```diff
@@ -7,6 +7,8 @@ Maintainer  : The Idris Community.
 -}
 
 {-# LANGUAGE DeriveFunctor, FlexibleContexts, PatternGuards #-}
+-- FIXME: {-# OPTIONS_GHC -fwarn-incomplete-patterns #-}
```

Comment:
Is it possible to not have the `FIXME`?

Reply:
I tried to introduce `-fwarn-incomplete-patterns` here but there were too many warnings. 😢 

I'm not sure what you mean by your comment. Is your preference that I remove the entire line with the FIXME or just the word FIXME?

Reply:
I think it would be best to avoid this line if possible.